Failure is intermittent enough that bisect is proving to be more hassle than it's worth. Seems pretty consistent in CI.

{noformat}
    [junit] Testcase: testDeleteIfNotDirty(org.apache.cassandra.db.CommitLogTest):      Caused an ERROR
    [junit] java.nio.file.AccessDeniedException: build\test\cassandra\commitlog;0\CommitLog-5-1431965988394.log
    [junit] FSWriteError in build\test\cassandra\commitlog;0\CommitLog-5-1431965988394.log
    [junit]     at org.apache.cassandra.io.util.FileUtils.deleteWithConfirm(FileUtils.java:131)
    [junit]     at org.apache.cassandra.io.util.FileUtils.deleteWithConfirm(FileUtils.java:148)
    [junit]     at org.apache.cassandra.db.commitlog.CommitLogSegmentManager.recycleSegment(CommitLogSegmentManager.java:360)
    [junit]     at org.apache.cassandra.db.commitlog.CommitLog.recover(CommitLog.java:166)
    [junit]     at org.apache.cassandra.db.commitlog.CommitLog.startUnsafe(CommitLog.java:416)
    [junit]     at org.apache.cassandra.db.commitlog.CommitLog.resetUnsafe(CommitLog.java:389)
    [junit]     at org.apache.cassandra.db.CommitLogTest.testDeleteIfNotDirty(CommitLogTest.java:178)
    [junit] Caused by: java.nio.file.AccessDeniedException: build\test\cassandra\commitlog;0\CommitLog-5-1431965988394.log
    [junit]     at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:83)
    [junit]     at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
    [junit]     at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
    [junit]     at sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:269)
    [junit]     at sun.nio.fs.AbstractFileSystemProvider.delete(AbstractFileSystemProvider.java:103)
    [junit]     at java.nio.file.Files.delete(Files.java:1126)
    [junit]     at org.apache.cassandra.io.util.FileUtils.deleteWithConfirm(FileUtils.java:125)
{noformat}